packages/app/src/components/promo/PromoHeader.vue

```diff
@@ -0,0 +1,50 @@
+<template>
+  <div class="flex flex-col bg-gray-50 border-gray-100 py-[16px] px-[24px] rounded-t-[8px] gap-16px items-start w-full">
+    <div class="promo-header flex flex-row w-full items-center flex-wrap gap-[16px]">
```

Comment:
```suggestion
    <div class="flex flex-row w-full items-center flex-wrap gap-[16px]">
```

Doesn't look like `promo-header` is used anywhere

Reply:
It's actually applying a class from the `style` block in this component - I didn't realize that Tailwind has support for [arbitrary box shadow declarations](https://tailwindcss.com/docs/box-shadow#arbitrary-values) though, so I'll refactor to use that